portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -14,19 +14,21 @@
 
 package com.google.sps.data;
 
-/** Class containing the author, content, and timestamp of a comment. */
+/** Class containing the author, content, timestamp, and image of a comment. */
 public final class Comment {
 
   private final long id;
   private final String content;
   private final String author;
   private final long timestamp;
+  private final String image;
```

Comment:
Could you add a comment on format - is it the raw pixel byte string or a URL?

Reply:
They are all URLs. Just fixed it!